framework/src/problems/FEProblemBase.C

```diff
@@ -8098,6 +8098,11 @@ FEProblemBase::addOutput(const std::string & object_type,
   const InputParameters * common = output_warehouse.getCommonParameters();
   if (common)
     parameters.applyParameters(*common, exclude);
+  if (common && std::find(exclude.begin(), exclude.end(), "execute_on") != exclude.end() &&
+      common->isParamSetByUser("execute_on"))
+    mooseInfoRepeated(
+        "'execute_on' parameter specified in [Outputs] block is ignored for object '" +
+        object_name + "'.\nDefine this object in its own sub-block of [Outputs].");
```

Comment:
Can't you paramInfo on the `output_type = true` param?

Reply:
not sure how that will be more helpful

this isnt a FEProblem param, it s params from the commonOutputWarehouse

Reply:
but doesn't this error come from something like

```
exodus = true
```

?

Reply:
I think only console and checkpoint ignore the execute on.

Reply:
I ll silence console. But it s useful for checkpoint

Reply:
should be good now

Reply:
output_type is not a parameter of FEProblemBase btw. It goes through an action then to the output_warehouse. so paramInfo is not straightforward

Reply:
it cant be a warning there are hundreds of instances to change
If a follow up to https://github.com/idaholab/moose/pull/26945 tackles mooseInfo this will be caught

Reply:
that's fine. I wasn't suggesting that it needed to be, was just describing capability.

probably worthwhile adding the file path/number to mooseInfo too